invisible_cities/cities/detsim_simulate_electrons_test.py

```diff
@@ -0,0 +1,142 @@
+import os
+import numpy  as np
+import tables as tb
+import pandas as pd
+
+from collections import namedtuple
+from operator    import itemgetter
+
+from pytest import fixture
+
+import invisible_cities.core.system_of_units as units
+
+from invisible_cities.io.mcinfo_io import load_mchits_df
+
+from invisible_cities.cities.detsim_simulate_electrons import generate_ionization_electrons
+from invisible_cities.cities.detsim_simulate_electrons import drift_electrons
+from invisible_cities.cities.detsim_simulate_electrons import diffuse_electrons
+
+@fixture(params = ["Kr83_nexus_v5_03_00_ACTIVE_7bar_3evts.MCRD.h5"])
+def nexusMC_filename(request, ICDATADIR):
+    return os.path.join(ICDATADIR, request.param)
+
+
+@fixture
```

Comment:
The default scope is `function`. I guess you don't want to execute the code of this fixture every time a test needs it, right? If so:
```suggestion
@fixture(scope="session")
```